client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -26,21 +26,10 @@ function ProfileInfoEdit () {
                             </Form.Text>
                         </Form.Group>
 
-                        <Row>
-                            <Col>
-                                <Form.Group controlId="formUserFirstName">
-                                    <Form.Label>First Name</Form.Label>
-                                    <Form.Control type="text" placeholder="John" />
-                                </Form.Group>
-                            </Col>
-                            
-                            <Col>
-                                <Form.Group controlId="formUserLastName">
-                                    <Form.Label>Last Name</Form.Label>
-                                    <Form.Control type="text" placeholder="Doe" />
-                                </Form.Group>
-                            </Col>
-                        </Row>
+                        <Form.Group controlId="formUserFirstName">
+                            <Form.Label>Research Group Name</Form.Label>
+                            <Form.Control type="text" placeholder="Team name.." />
```

Comment:
I know this isn't really related to this PR, but if you're going to make changes to it regarding other changes requested, could we add the current value in place of the placeholders? So using something like `const teamName = useSelector(state => state.user?.teamName)` we can get the team name from the Redux store and then in place of the `Team name...` place holder we can probably use `${teamName}`. And this relates to the country and phone number as well.

Reply:
Followed suggestion to use Redux to fetch value from state to fill in placeholders.